### PR TITLE
Create update-image-builder.yml

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

File: .github/workflows/update-image-builder.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  update-image-builder:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Push commit to image-builder repo
+        run: |
+          git clone "https://${{ secrets.PROW_BOT_TOKEN }}@github.com/open-cluster-management/image-builder.git"
+          cd image-builder
+          echo "$GITHUB_REPOSITRY@%GITHUB_SHA" > .trigger-openshift-ci"
```
Comment (schmidtd) [inline .github/workflows/update-image-builder.yml:15]: Should `$GITHUB_REPOSITRY@%GITHUB_SHA` be `$GITHUB_REPOSITRY@$GITHUB_SHA`?

Comment (ray-harris) [reply .github/workflows/update-image-builder.yml:15]: $GITHUB_REPOSITORY :)

Comment (ray-harris) [reply .github/workflows/update-image-builder.yml]: Oh. Now I see the percent sign in front of GITHUB_SHA. So there were two issues.

**Review by schmidtd — APPROVED** [review]
/lgtm

**Pull request merged**